src/app.controller.ts

```diff
@@ -1,8 +1,8 @@
 import { Controller, Get } from '@nestjs/common';
-import { AppService } from './app.service';
+import AppService from './app.service';
 
 @Controller()
-export class AppController {
+export default class AppController {
```

Comment:
Default exports are not recommended with Nest and should be permitted rather than suggested

Reply:
The rule being followed here is https://airbnb.io/javascript/#modules--prefer-default-export.